service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/FurtherEvidenceUploadedEventHandler.java

```diff
@@ -823,4 +826,18 @@ public void notifyTranslationTeam(FurtherEvidenceUploadedEvent event) {
                 bundle.getValue().getDocument(), bundle.getValue().asLabel())
             );
     }
+
+    @EventListener
+    public void createWorkAllocationTask(FurtherEvidenceUploadedEvent event) {
+        CaseData caseData = event.getCaseData();
+        CaseData caseDataBefore = event.getCaseDataBefore();
+
+        if (!getNewCorrespondenceDocumentsByLA(caseData, caseDataBefore).getDocumentReferences().isEmpty()
+            || !getNewCorrespondenceDocumentsBySolicitor(caseData, caseDataBefore).getDocumentReferences().isEmpty()
+            || !getNewCorrespondenceDocumentsByHmtcs(caseData, caseDataBefore).getDocumentReferences().isEmpty()
+        ) {
+            log.info("Creating CORRESPONDENCE_UPLOADED work allocation task for case: {}", caseData.getId());
```

Comment:
Think this one we can remove as we're logging it in the `createWorkAllocationTask`